src/autopas/containers/linkedCells/traversals/C04SoACellHandler.h

```diff
@@ -211,13 +211,13 @@ inline void C04SoACellHandler<ParticleCell, PairwiseFunctor, dataLayout, useNewt
         auto stripeView = cell1->_particleSoABuffer.constructView(0, numParticlesBaseCell);
         if (slice == currentSlice) {
           // process stripe with itself
-          _pairwiseFunctor->SoAFunctor(stripeView, useNewton3);
+          _pairwiseFunctor->SoAFunctorSingle(stripeView, useNewton3);
```

Comment:
@HobbyProgrammer does the C04SoATraversal pass a view to the SoAFunctor that potentially has owned AND non-owned particles?

Reply:
also, check the other calls to SoAFunctorPair, please :)

Reply:
@SteffenSeckler I don't know, I have never looked deeper into how this traversal works. It was just ported to `SoAView`s from a previous implementation of views in the `SoA`. @Garfield96 maybe knows!

Reply:
push @Garfield96

Reply:
@SteffenSeckler Potentially yes. This traversal does not distinguish between owned and non-owned (at least it didn't then I implemented it)

Reply:
ok. thanks for the info.